lib/src/io_client.dart

```diff
@@ -3,37 +3,24 @@
 // BSD-style license that can be found in the LICENSE file.
 
 import 'dart:async';
+import 'dart:io';
```

Comment:
Does this mean "package:http/http.dart" can only be imported on the command line VM? If so, isn't that a breaking change?